eclass/fcaps.eclass

```diff
@@ -33,15 +34,12 @@ _FCAPS_ECLASS=1
 
 IUSE="+filecaps"
 
-# Since it is needed in pkg_postinst() it must be in RDEPEND
+# Since it is needed in pkg_postinst() it must be in IDEPEND
 case "${EAPI:-0}" in
-	[0-6])
-		RDEPEND="filecaps? ( sys-libs/libcap )"
-	;;
-	*)
-		BDEPEND="filecaps? ( sys-libs/libcap )"
-		RDEPEND="${BDEPEND}"
-	;;
+	7) BDEPEND="filecaps? ( sys-libs/libcap )" ;&
```

Comment:
I have never seen `;&` before. That's neat!

Reply:
But this means the eclass will throw a syntax error on bash-3.2, instead of giving unsupported EAPI die.

I think the best way would be to split the `case` into a more traditional ebuild thingie, i.e.:

```
case ${EAPI} in
    [6-8]) ;;
    *) die "EAPI ${EAPI:-0} is unsupported";;
esac

case ${EAPI} in
    7) ...;&
    6) ...;;
    *) ...;;
esac
```

Reply:
also, put the whitelist before the inherit guard, and the `IDEPEND` assignment inside the inherit guard. Finally, maybe consider

```sh
case ${EAPI} in
    7) ...;& # fallthrough
    6) ...;;
    *) ...;;
esac
```

so people don't get triggered?